ansible/roles/debops.phpipam/defaults/main.yml

```diff
@@ -10,240 +10,365 @@
+phpipam__src: '{{ (ansible_local.root.src
                   if (ansible_local|d() and ansible_local.root|d() and
                       ansible_local.root.src|d())
-                  else "/usr/local/src") + "/" + phpipam_user }}'
+                  else "/usr/local/src") + "/" + phpipam__user }}'
 
                                                                    # ]]]
-# .. envvar:: phpipam_www [[[
+# .. envvar:: phpipam__www [[[
 #
 # Base web root directory for phpIPAM website
-phpipam_www: '{{ (ansible_local.nginx.www if (ansible_local is defined and ansible_local.nginx is defined and ansible_local.nginx.www is defined) else "/srv/www") + "/" + phpipam_user }}'
+phpipam__www: '{{ (ansible_local.nginx.www
+                  if (ansible_local is defined and
+                      ansible_local.nginx is defined and
+                      ansible_local.nginx.www is defined)
+                  else "/srv/www") + "/" + phpipam__user }}'
 
                                                                    # ]]]
-# .. envvar:: phpipam_webserver_user [[[
-#
-# phpIPAM webserver user (needs read-only access to the website code)
-phpipam_webserver_user: '{{ ansible_local.nginx.user if (ansible_local is defined and ansible_local.nginx is defined and ansible_local.nginx.user is defined) else "www-data" }}'
-
                                                                    # ]]]
-# ---- git repository and checkout ----
+# Git repository and checkout [[[
+# -------------------------------
 
-# .. envvar:: phpipam_git_repo [[[
+# .. envvar:: phpipam__git_repo [[[
 #
 # phpIPAM source repository
-phpipam_git_repo: 'https://github.com/debops/phpipam.git'
+phpipam__git_repo: 'https://github.com/phpipam/phpipam.git'
 
                                                                    # ]]]
-# .. envvar:: phpipam_git_dest [[[
+# .. envvar:: phpipam__git_dest [[[
 #
 # phpIPAM source directory on the host
-phpipam_git_dest: '{{ phpipam_src + "/" + phpipam_git_repo.split("://")[1] }}'
+phpipam__git_dest: '{{ phpipam__src + "/" + phpipam__git_repo.split("://")[1] }}'
 
                                                                    # ]]]
-# .. envvar:: phpipam_git_version [[[
+# .. envvar:: phpipam__git_version [[[
 #
 # phpIPAM git branch to deploy
-phpipam_git_version: 'master'
+phpipam__git_version: '1.3.2'
 
                                                                    # ]]]
-# .. envvar:: phpipam_git_checkout [[[
+# .. envvar:: phpipam__git_checkout [[[
 #
 # Default path where phpIPAM source files will be deployed
-phpipam_git_checkout: '{{ phpipam_www + "/sites/" + phpipam_domain[0] + "/public" }}'
+phpipam__git_checkout: '{{ phpipam__www + "/sites/" + phpipam__fqdn + "/public" }}'
 
                                                                    # ]]]
-# ---- Other variables ----
+                                                                   # ]]]
+# phpIPAM config.php [[[
+# -------------------
```

Comment:
The underline needs to be the same length as the title, otherwise Sphinx rendering will break.